src/PokemonContainer.js

```diff
@@ -0,0 +1,123 @@
+import { Component } from 'react';
+import PokemonList from './PokemonList';
+import Loader from "react-loader-spinner";
+
+
+import './App.css';
+
+class PokemonContainer extends Component {
+  state = { data: [], loading: true, query: null, sortOrder: 'asc', page: 1 };
```

Comment:
Remember that your default data should always be the same data TYPE as the data that you plan to put into storage. This is where you are initializing your data so query, in this case, should be an empty string since the end data type is also a string.